docs/architecture/whitelisting.md

```diff
@@ -75,3 +76,60 @@ the following procedure:
 After these steps are done, the ERC20 whitelisting is finished. Liquidity for
 the given asset should promptly begin deposits in order to provide a smooth
 experience for users interested in interacting with the token.
+
+## Detailed Procedure
+### Prerequisites for Submission
+
+Before initiating a governance proposal to whitelist an ERC-20 token as a ZRC-20, ensure you meet the following prerequisites:
+
+* **Zetacored CLI Installation**: Make sure the zetacored CLI tool is installed on your system. This tool is essential for interacting with the Zetachain network and submitting proposals.
+* **Wallet Setup and Funding**: A wallet must be set up and associated with the zetacored CLI. This wallet should contain a minimum of 1000 ZETA tokens, plus additional ZETA for gas fees, to cover the deposit required for proposal submission.
+
+### **Submitting a Governance Proposal**
+
+To submit a governance proposal for whitelisting an ERC-20 token, follow these steps:
+
+1.  **Prepare the Proposal JSON File**: Draft a `mainnet_proposal.json` file that outlines the details of the token you wish to whitelist. This file must include:
```

Comment:
We already have governance docs that [cover text proposals](https://www.zetachain.com/docs/developers/cli/governance/text/). Any chance we can incorporate some of these changes there and link to these documents from this one?

Reply:
The text proposals are generalized vs . this one coves the spefic example of listing an ERC20